### DISCUSSION TITLE
JsonTable Spaltenüberschrift ändern

### DISCUSSION THREAD

**Discussion by majornaese** [Zefau/ioBroker.jarvis#1100] [Q&A]
Hallo zusammen,
gibt es eine Möglichkeit bei einem JsonTable die Spaltenüberschriften unabhängig von den Bezeichnern im JSON anzugeben? Also wenn das Element im JSON "createdDate" heißt, dann hätte ich gerne den Spaltennamen "Datum Erstellung".
Ich habe dazu nichts gefunden. 
Danke!

**Comment by Zefau** [comment]
In v3 ja. Siehe https://quasar.dev/vue-components/table#defining-the-columns für mögliche Optionen.

**Comment by Zefau** [comment]
Siehe https://github.com/Zefau/ioBroker.jarvis/issues/1060#issuecomment-932884490 als Beispiel

**Comment by majornaese** [comment]
Danke für die schnelle Rückmeldung @Zefau !
Wie kommt man denn an die v3? Im latest-Repository ist die v2.2.3 aktuell. Würden beim Update auf v3 meine Layouts und Geräte erhalten bleiben?

**Reply by Zefau** [reply]
v3 ist aktuell noch alpha und nur über Github erhältlich, da noch nicht stabil. Geräte bleiben erhalten; Layout wird konvertiert (da grundlegend überarbeitet).

**Comment by majornaese** [comment]
Danke! Update hat funktioniert, sieht alles richtig nice aus! Und die neuen Tabellenfeatures sind gut gelungen!
